### PR TITLE
Created a linkfree portfolio

### PR DESCRIPTION
**Pull request by Ewaaugustine**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4157"></a>

### CODE REVIEW

File: data/Ewaaugustine.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Ewa Augustine",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student Doctor turned **Software Developer**. Lover of good things and a technical writer",
+  "avatar": "https://github.com/Ewaaugustine.png"
```
Comment (swagataroy30) [inline data/Ewaaugustine.json:6]:
```suggestion
  "avatar": "https://github.com/Ewaaugustine.png",
```

**Review by swagataroy30 — COMMENTED** [review]
You need to add a comma at line 6. Thank You!

File: data/Ewaaugustine.json (hunk 1 above)
Comment (kumarsonsoff3) [inline data/Ewaaugustine.json:6]: Hey @Ewaaugustine, as of LinkFree's latest update, we no longer need the `avatar` key. So please remove this avatar field from your file. 

[Updated docs](https://linkfree.eddiehub.io/docs/quickstart)
```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/Ewaaugustine.json (hunk 1 above)
Comment (kumarsonsoff3) [reply data/Ewaaugustine.json:6]: We no longer need the `avatar` key as of our latest update.

**Comment by Ewaaugustine** [comment]
Good day from here,
I'm reaching out with respect to the previous mail sent to making
corrections.
If I remove the avatar and add comma my contribution will be accepted right?

Or is there any other changes expected of me to be made?

**Comment by kumarsonsoff3** [comment]
Great day to you too, 

You just need to remove the avatar field as suggested in the latest mail, no comma is needed. (Simply just remove line no. `6`). Should be good to go. 

Previous one to add the comma is now outdated, as of Linkfree's latest update. :)

**Comment by Ewaaugustine** [comment]
Good day from here,
Please can you check my linkfree correction let me if i did the right thing.

Also, i thought that i'll get my own linkfree link sent to my email if my
contribution went through?

**Comment by eddiejaoude** [comment]
@Ewaaugustine it looks like you deleted your fork and therefore this closed the pull request. Please open a new pull request with your profile and we can get it reviewed and merged

I think you It looks like you have deleted the fork or branch in your fork twice now, and this keeps closing your pull request https://github.com/EddieHubCommunity/LinkFree/pulls?q=is%3Apr+author%3AEwaaugustine+is%3Aclosed

**Pull request closed without merging**